pkg/agent/datamodel/sig_config.go

```diff
@@ -239,7 +239,7 @@ const (
 
 	Arm64LinuxSIGImageVersion    string = "2022.09.27"
 	Ubuntu2204SIGImageVersion    string = "2022.09.19"
```

Comment:
can you also please bump this up to 2022.09.27? this wasnt updated because the 2 image versions are out of sync.

Reply:
similarly, please update the tests for 22.04 in sig_config_test.go